docs/iris/src/developers_guide/documenting/whats_new_contributions.rst

```diff
@@ -23,27 +23,41 @@ contain multiple changes that are worth highlighting as contributions to the
 what's new document.
 
 Each contribution will ideally be written as a single concise bullet point
-in a reStructuredText format with a trailing blank line.  For example::
+in a reStructuredText format. Where possible do not exceed **column 80** and
+ensure that any subsequent lines of the same bullet point are aligned with the
+first. The content should target an Iris user as the audience. The required
+content, in order, is as follows:
 
-  * Fixed :issue:`9999`.  Lorem ipsum dolor sit amet, consectetur adipiscing 
-    elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. 
-    <blank line>
+* Names of those who contributed the change e.g. ``Robert FitzRoy and Luke
```

Comment:
An alternative way of doing this could be to use the names contributors use publicly on GitHub (and link to their profile page), as done e.g. here: https://docs.dask.org/en/latest/changelog.html#id16. That has the advantage that it's up to contributors themselves to decide if they want to be listed with their real name.

Reply:
Thanks @bouweandela , I had considered this but I only checked Cartopy, SciPy and Numpy so I hadn't seen any precedent. Good to know that Dask do it - I will make the change

Reply:
596f31b7aa678cecb4903b0898cbbeef73aba594